src/main/java/org/mds/pacman/Pacman.java

```diff
@@ -0,0 +1,55 @@
+package org.mds.pacman;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class Pacman {
+
+    private final AtomicInteger x;
+    private final AtomicInteger y;
+    private Direction direction;
+
+    public Pacman(AtomicInteger x, AtomicInteger y) {
+        this.x = x;
+        this.y = y;
+        direction = Direction.RIGHT;
+    }
+
+
+    public void modifyCoordinatesBasedOnDirection() {
+        switch (direction) {
+            case UP -> moveUp();
+            case DOWN -> moveDown();
+            case RIGHT -> moveRight();
+            case LEFT -> moveLeft();
+        }
+    }
+

```

Comment:
It probably can't be like this (plus minus 10), should be changed to 1 and decreased interval